src/Admin/Controller/ForumController.php

```diff
@@ -29,9 +29,21 @@ public function __invoke(
             : null;
 
         if ($form !== null) {
+            $originalGroup = $forum->getGroup();
             $form->handleRequest($request);
             if ($form->isSubmitted() && $form->isValid()) {
                 $updated = $form->getData();
+                $newGroup = $updated->getGroup();
+
+                if ($originalGroup !== $newGroup) {
+                    if ($newGroup !== null) {
```

Comment:
what happens when you remove the group from a forum?

Reply:
Removing the group would reset position to 0 and no checks conducted to make it increment. Removed the if and else.